cms/static/js/views/modals/course_outline_modals.js

```diff
@@ -275,12 +353,18 @@ define(['jquery', 'backbone', 'underscore', 'gettext', 'js/views/baseview',
         notTimedExam: function (event) {
             event.preventDefault();
             this.$('#id_time_limit_div').hide();
+            this.$('#id_review_rules_div').hide();
             this.$('#id_time_limit').val('00:00');
         },
         showTimeLimit: function (event) {
             event.preventDefault();
             this.$('#id_time_limit_div').show();
             this.$('#id_time_limit').val("00:30");
```

Comment:
Although not done in this branch, the time is reset to 00:30 every time we shift form timed to proctored to practice.
better to use:
`            if (!this.isValidTimeLimit(this.$('#id_time_limit').val())) {
                $(event.currentTarget).val("00:30");
            }
`